### PR TITLE
MRG, DOC: update SSS / Maxfilter tutorial

### PR DESCRIPTION
**Pull request by drammock**
also adds HPI to glossary

### CODE REVIEW

File: tutorials/preprocessing/plot_60_maxwell_filtering_sss.py [hunk 1]
```diff
@@ -1,45 +1,197 @@
 """
 .. _tut-artifact-sss:
 
-=======================================
-Artifact correction with Maxwell filter
-=======================================
+Signal-space separation and Maxwell filtering
```
Comment (larsoner) [inline tutorials/preprocessing/plot_60_maxwell_filtering_sss.py]:
```suggestion
Signal-space separation (SSS) and Maxwell filtering
```

File: tutorials/preprocessing/plot_60_maxwell_filtering_sss.py [hunk 2]
```diff
@@ -1,45 +1,197 @@
...
+#
+#
+# Using SSS and Maxwell filtering in MNE-Python
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# For optimal use of SSS with data from Elekta Neuromag® systems, you should
+# provide the path to the fine calibration file (which encodes site-specific
+# information about sensor orientation and calibration) as well as a crosstalk
+# compensation file (which reduces interference between Elekta's co-located
+# magnetometer and paired gradiometer sensor units.
```
Comment (larsoner) [inline tutorials/preprocessing/plot_60_maxwell_filtering_sss.py]:
```suggestion
# magnetometer and paired gradiometer sensor units).
```

File: tutorials/preprocessing/plot_60_maxwell_filtering_sss.py [hunk 3]
```diff
@@ -1,45 +1,197 @@
...
+# frequency of :math:`\frac{1}{\mathtt{st\_duration}}~\mathrm{Hz}`; this
+# effective high-pass has some important consequences:
+#
+# - In general, larger values of ``st_duration`` are better (provided that your
+#   computer has sufficient memory) because larger values of ``st_duration``
+#   will have a smaller effect on the signal.
+#
+# - It is best to choose a chunk duration that evenly divides your data length,
+#   so that the effective high-pass filtering is the same for all spans of your
+#   data.
```
Comment (larsoner) [inline tutorials/preprocessing/plot_60_maxwell_filtering_sss.py]: I'm not sure we really need to recommend this. If you have multiple subjects and/or runs, the only way to really satisfy this is to have the duration vary across subjects and runs, which is probably a worse tradeoff than having just the last chunk for each subject/run possibly be different (at least for typical duration recordings and `st_duration` choices).

Perhaps instead we should just make a note that the last window could use a longer duration (at worse just shy of `2 * st_duration`)?

File: tutorials/preprocessing/plot_60_maxwell_filtering_sss.py [hunk 4]
```diff
@@ -1,45 +1,197 @@
...
+# of electromagnetic fields. SSS separates the measured signal into components
+# attributable to sources *inside* the measurement volume of the sensor array
+# (the *internal components*), and components attributable to sources *outside*
+# the measurement volume (the *external components*). The internal and external
+# components are linearly independent, so it is possible to simply discard the
+# external components to reduce environmental noise. *Maxwell filtering* is a
+# related procedure that omits the higher-order components of the internal
+# subspace, which are dominated by sensor noise. Typically, Maxwell filtering
+# and SSS are performed together (in MNE-Python they are implemented together
+# in a single function).
```
Comment (larsoner) [inline tutorials/preprocessing/plot_60_maxwell_filtering_sss.py]: I actually didn't know there was a distinction (but there very well could be)! Is this adapted from the Maxfilter manual, or somewhere else?

File: tutorials/preprocessing/plot_60_maxwell_filtering_sss.py [hunk 5]
```diff
@@ -1,45 +1,197 @@
...
+#
+# Like :ref:`SSP <tut-artifact-ssp>`, SSS is a form of projection. Whereas SSP
+# empirically determines a noise subspace based on data (empty-room recordings,
+# EOG or ECG activity, etc) and projects the measurements onto a subspace
+# orthogonal to the noise, SSS mathematically constructs the external and
+# internal subspaces from `spherical harmonics`_ and reconstructs the sensor
+# signals using only the internal subspace.
```
Comment (larsoner) [inline tutorials/preprocessing/plot_60_maxwell_filtering_sss.py]:
```suggestion
# signals using only the internal subspace (i.e., does an oblique projection).
```

File: tutorials/preprocessing/plot_60_maxwell_filtering_sss.py (hunk 4 above)
Comment (drammock) [reply tutorials/preprocessing/plot_60_maxwell_filtering_sss.py]: > This process of leaving out the high *l*, *m* end of the spectrum not only reduces the noise like ordinary spatial filtering but also guarantees that the retained field distribution is consistent with Maxwell's equations ∇ ⋅ **b** = 0 and the quasistatic ∇ ⨯ **b** = 0 in empty space. Therefore, we call this procedure "Maxwell filtering."

Taulu, S., Simola, J., & Kajola, M. (2005). Applications of the signal space separation method. IEEE Transactions on Signal Processing, 53(9), 3359–3372. https://doi.org/10.1109/TSP.2005.853302

File: tutorials/preprocessing/plot_60_maxwell_filtering_sss.py (hunk 3 above)
Comment (drammock) [reply tutorials/preprocessing/plot_60_maxwell_filtering_sss.py]: LMK if you're happy with the revised wording.

**Comment by drammock** [comment]
revised rendering: https://14632-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/preprocessing/plot_60_maxwell_filtering_sss.html

**Comment by drammock** [comment]
@larsoner ready for merge once the CIs come back green (only error last time was CodeSpell).

**Comment by larsoner** [comment]
Thanks @drammock

**Pull request merged**